### ISSUE TITLE
Cannot Randomize all paramters at once

### ISSUE THREAD

**Issue by ldslds1992** [kernicPanel/bitwig-randomizer#17]
Hi! Really love your script. The randomize all paramter function doenst work for me (Bitwig 5.1). It works the same as the "randomize current page" function. Any idea whta could be the problem? Thank in advance!

**Comment by markrhysmitchell** [comment]
I've noticed this problem too. It's a bummer, as this is pretty much my favourite Bitwig script, but it might not be getting updates anymore, I'm afraid.

**Comment by kernicPanel** [comment]
Hi,
Sorry for not updating this script. I didn't work on it for quite a long time.
But I'll look for fixing this when possible. 
Thanks for your interest and your patience.

**Comment by markrhysmitchell** [comment]
Of course, there's nothing to apologise for - I just assumed you'd moved on from it! But if you are able to update it for the new script system, that would be amazing. It really is a fantastic creation.
